flask-deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todo-app-deployment
+  labels:
+    app: todo-app
+spec:
+  replicas: 1
+  selector:
```

Comment:
@Arun07L what is the purpose of these labels?

Reply:
Generally App identification ku use pantrom.Metadata la irruka label deployment la app identification ku and template la iruka label pod ku. service la label name use panni antha label la entha pod running akitu iruko anth pod ah target panna mudiyum.